packages/aws-cdk-lib/aws-bedrock/lib/foundation-model.ts

```diff
@@ -104,6 +104,9 @@ export class FoundationModelIdentifier {
   /** Base model "anthropic.claude-3-sonnet-20240229-v1:0:200k" */
   public static readonly ANTHROPIC_CLAUDE_3_SONNET_20240229_V1_0_200K = new FoundationModelIdentifier('anthropic.claude-3-sonnet-20240229-v1:0:200k');
 
+  /** Base model "anthropic.claude-3-5-sonnet-20240620-v1:0" */
+  public static readonly ANTHROPIC_CLAUDE_3_5_SONNET_20240620_V1_0_200K = new FoundationModelIdentifier('anthropic.claude-3-5-sonnet-20240620-v1:0');
```

Comment:
Is it a mistake?

```suggestion
  public static readonly ANTHROPIC_CLAUDE_3_5_SONNET_20240620_V1_0 = new FoundationModelIdentifier('anthropic.claude-3-5-sonnet-20240620-v1:0');
```

Reply:
@go-to-k 
Thanks.(Silly mistake...😅
Fixed it.